includes/installer/initSettings.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/* 
+ * dashboard for admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+// Connect to DB
+//
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Error connecting database. Please check your database credentials and update.";
+	die();
+}
+
+
+// admin settings table
+
+$query = "CREATE TABLE IF NOT EXISTS $settings ( $setKey VARCHAR(255) NOT NULL, $setValue VARCHAR(10) NOT NULL, INDEX idx USING BTREE ($setKey) )";
+
+$stmt = $sqlConn->prepare( $query );
+
+
+if ( ! $stmt->execute() ) // if unsuccessful
+{
+	echo "Error creating settings database.";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->